Update FAQ.md (#9587)
iptv-org/iptv: FAQ.md

* A PC with Internet, video editor and recording/broadcasting app under RTMP protocol (e.g. OBS).
* And a live streaming service with multibitrate (if possible).

> After you have your m3u8 link, add your channel information creating or editing the id (identifier) onto the [Database](https://github.com/iptv-org/database) repository. Then, submit an [issue](https://github.com/iptv-org/iptv/issues/new/choose) to propose it to us.

Reviewer: ```suggestion
After you have your m3u8 link, add your channel information by creating or editing the id (identifier) onto the [Database](https://github.com/iptv-org/database) repository. Then, submit an [issue](https://github.com/iptv-org/iptv/issues/new/choose) to propose it to us.
```

Notes:

> * Because this is an community repository, we need some approvals from the administrators to continue. See [discussion](https://github.com/iptv-org/iptv/discussions/8412).

Reviewer: ```suggestion
* Because this is an community repository, we need the approval from two maintainers to accept your new additions. See [discussion](https://github.com/iptv-org/iptv/discussions/8412) for further explanations.
```

* If your television station broadcasts from other television stations, please read the *[Can I safely add an FTA channel that broadcasts PPV events or pay programming?](#Can-i-safely-add-an-fta-channel-that-broadcasts-ppv-events-or-pay-programming)* section.

> * The same id can only be used if a mirror stream is also proposed. Please do not set the same id for two different channels. See [issue](https://github.com/iptv-org/iptv/discussions/8412).

Reviewer: ```suggestion
* The same id can only be used if a mirror stream is also proposed. Please do not set the same id for two different channels.
```
I don't see why #8412 can give any explanations to this note.

Author: Sorry. The link is https://github.com/iptv-org/iptv/pull/9572.
